web/src/main/resources/org/springframework/security/web/messages_en.properties

```diff
@@ -0,0 +1 @@
+ExceptionTranslationFilter.insufficientAuthentication=Full authentication is required to access this resource
```

Comment:
Add this key to messages.properties (default bundle) as well

Reply:
Done!